src/containers/Layout.tsx

```diff
@@ -41,6 +41,13 @@ const Layout = ({ children, withoutNavigation }: LayoutProps) => {
             <li>
               <Link to="/">Home</Link>
               <Link to="/about">About</Link>
+              <Link to="/explore">Explore</Link>
+              <Link to="/notifications">Notifications</Link>
+              <Link to="/messages">Messages</Link>
+              <Link to="/bookmarks">Bookmarks</Link>
+              <Link to="/lists">Lists</Link>
+              <Link to="/profile">Profile</Link>
+              <Link to="/more">More</Link>
```

Comment:
I think it's not required if you already added navigation links in NavigationBar